### PR TITLE
v0.28.0 — Updated DangerJS config.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Changed
- Updated DangerJS config.

### CODE REVIEW

**Comment by DamianMullins** [comment]
Bit confused about the DangerJS warning, the version has been updated? @ashleynolan any ideas?

**Comment by ashleynolan** [comment]
@DamianMullins Looks like it's an issue with the package diff on line 23:

```
const packageDiff = danger.git.JSONDiffForFile('package.json');
```

The logs are reporting that there is a change, but for some reason the version number is empty.  You can run danger locally so could take a look at why that might be by doing that and adding some logs?

**Comment by DamianMullins** [comment]
Gah, v0.28.0 hasn't been released, that's the issue. I'll fix now.

**Comment by DamianMullins** [comment]
Oh no, it's actually me pushing things which shouldn't have been pushed 🆘

File: dangerfile.js [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
-
     // Check for version update
     const hasPackageJsonChanged = danger.git.modified_files.includes('package.json');
     const packageDiff = danger.git.JSONDiffForFile('package.json');
 
     packageDiff.then(result => {
         if (!hasPackageJsonChanged || (hasPackageJsonChanged && !result.version)) {
             const semverLink = 'https://docs.npmjs.com/getting-started/semantic-versioning';
-            /* eslint-disable no-console */console.log('Versioning Missing'); console.log(hasPackageJsonChanged, result);/* eslint-enable no-console */
+            console.log('Versioning Missing');
```
Comment (kevinrodrigues) [inline dangerfile.js]: Worth just making one console log call here instead of two?

Comment (DamianMullins) [reply dangerfile.js]: I think the aim was to have them log out on separate lines?

Comment (kevinrodrigues) [reply dangerfile.js]: Coolio.

**Pull request merged**